src/SFA.DAS.EmployerIncentives.Web/ViewModels/Apply/SelectApprenticeships/SelectApprenticeshipsViewModel.cs

```diff
@@ -2,27 +2,22 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace SFA.DAS.EmployerIncentives.Web.ViewModels.Apply
+namespace SFA.DAS.EmployerIncentives.Web.ViewModels.Apply.SelectApprenticeships
 {
     public class SelectApprenticeshipsViewModel : ViewModel
     {
         public const string SelectApprenticeshipsMessage = "Select the apprentices you want to apply for";
 
-        public SelectApprenticeshipsViewModel() : base(SelectApprenticeshipsMessage)
-        {
-            SelectedApprenticeships = new List<string>();
-        }
+        public SelectApprenticeshipsViewModel() : base(SelectApprenticeshipsMessage) { }
 
-        public IEnumerable<ApprenticeshipModel> Apprenticeships { get; set; }
-
-        public bool HasSelectedApprenticeships => SelectedApprenticeships.Count > 0;
+        public string AccountId { get; set; }
 
-        public string FirstCheckboxId => $"new-apprenticeships-{Apprenticeships.First().Id}";
+        public string AccountLegalEntityId { get; set; }
 
-        public List<string> SelectedApprenticeships { get; set; }
+        public IEnumerable<ApprenticeshipModel> Apprenticeships { get; set; }
 
-        public string AccountId { get; set; }
+        public string FirstCheckboxId => $"new-apprenticeships-{Apprenticeships.First().Id}";
```

Comment:
Is this needed?  it's not used in the view so shouldn't be here.

Reply:
it *is* used in the view for error handling.